### PR TITLE
Feature/128097 mobile month view events

### PR DESCRIPTION
**Pull request by paulmskim**
For ticket: https://central.tri.be/issues/128097

### CODE REVIEW

File: src/views/v2/month.php [hunk 1]
```diff
@@ -15,12 +15,24 @@
...
 
-$this->template( 'top-bar' );
+<div class="tribe-common-l-container tribe-events-l-container">
 
-?>
+	<?php $this->template( 'events-bar' ); ?>
+
+	<?php $this->template( 'top-bar' ); ?>
+
+	<div class="tribe-events-calendar-month">
```
Comment (ryanurban) [inline src/views/v2/month.php]: Are we missing the aria grid stuff here, or will this be getting merged with the work Juan has done on this view?

File: src/views/v2/month/mobile-event.php [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+$classes = [ 'tribe-events-calendar-month__mobile-event' ];
+
+/* @todo fix this once we make event dynamic */
+// if ( tribe( 'tec.featured_events' )->is_featured( $event_id ) ) {
+	$classes[] = 'tribe-events-calendar-month__mobile-event--featured';
+// }
+?>
+
+<article class="<?php echo esc_attr( implode( ' ', $classes ) ); ?>">
```
Comment (ryanurban) [inline src/views/v2/month/mobile-event.php]: Do we need aria grid atts for the mobile implementation or not necessary?

File: src/views/v2/month/mobile-event/featured-image.php [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+ *
+ * @link {INSERT_ARTCILE_LINK_HERE}
+ *
+ * @version TBD
+ *
+ */
+?>
+<div class="tribe-events-calendar-month__mobile-event-featured-image-wrapper">
+	<div class="tribe-events-calendar-month__mobile-event-featured-image tribe-common-c-image tribe-common-c-image--bg">
+		<div class="tribe-common-c-image__bg"></div>
```
Comment (ryanurban) [inline src/views/v2/month/mobile-event/featured-image.php]: For any bgd image implementations, assuming they aren't decoration, but actual images for the event, we should be sure to add a `role="img" aria-label="alt text here"` on your bg image container.

File: src/views/v2/month/mobile-event.php (hunk 2 above)
Comment (paulmskim) [reply src/views/v2/month/mobile-event.php]: Nope, this is one of the events that shows up below the calendar when a day is pressed.

File: src/views/v2/month/mobile-event/featured-image.php (hunk 3 above)
Comment (paulmskim) [reply src/views/v2/month/mobile-event/featured-image.php]: gotcha, will do!

**Comment by lucatume** [comment]
I've merged more test fixes from `release/B19.07` to make it pass tests.

**Pull request merged**